AirCasting/Dashboard/AirSectionPickerView.swift

```diff
@@ -63,7 +63,7 @@ struct PickerButtonStyle: ButtonStyle {
         configuration
             .label
             .foregroundColor(isSelected ? Color.accentColor : Color.aircastingGray)
-            .font(Fonts.muliHeading2)
+            .font(Fonts.muliRegularHeading3)
```

Comment:
In designs the font is semibold for selected section

Reply:
I tested the change but SwiftUI animates this change and it looks weird to me. What do you think?

https://user-images.githubusercontent.com/82096200/177705495-83e62eb5-9553-4e6e-83eb-b765b39c3832.mov